tests/main/security-dev-input-event-denied/task.yaml

```diff
@@ -50,7 +50,7 @@ execute: |
         exit 1
     fi
     # AppArmor is 'Permission denied' which is expected with default policy
-    MATCH "Permission denied" < call.error
+    MATCH "(Permission denied|Operation not permitted)" < call.error
```

Comment:
This is unexpected. When AppArmor denies an operation it returns EACCESS (Permission denied), but in this case we're getting EPERM, suggesting that the call was blocked for some other reason. I don't think we should do this change without understanding why the error is different.

Reply:
@bboozzoo, thanks for the review, agree with you, I'll run this in a 20.04 and compare the dmesg info, perhaps that could help.

Reply:
@bboozzoo I found this: https://paste.ubuntu.com/p/jVYtFCvbgP/

Reply:
@sergiocazzolato actually the messages / returned error messages from that pastebin make sense and are correct, when the joystick interface is not connected, the first error message should be "Permission Denied", and the second error message when the joystick interface _is connected_ should be "Operation not permitted"

Reply:
when this test fails, what I think it indicates is that either:
* the apparmor message for the failure is wrong and for some reason appamor is causing EPERM due to some bug there
* the generated policy is wrong and we are somehow allowing access to whatever device node is referenced by the `-event-kbd` argument to evdev
* the device node referenced by the `-event-kbd` argument to evdev actually shows up somewhere else, and is not what we think it is anymore (i.e. maybe it's not /dev/input/event* but it's /dev/event* or some other odd thing